test/src/main/webapp/META-INF/context.xml

```diff
@@ -1,16 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Context>
 	<Resource
-		name="jdbc/ibis4test"
-		type="javax.sql.DataSource"
-		username="testiaf_user"
+		name="jdbc/ibis4test-h2"
+		type="org.h2.jdbcx.JdbcDataSource"
+		factory="org.apache.naming.factory.BeanFactory" 
+		URL="jdbc:h2:file:./ibis4test;MODE=Oracle;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE"
+	/>
+	<Resource
+		name="jdbc/ibis4test-oracle"
+		factory="org.apache.naming.factory.BeanFactory"
+		type="oracle.jdbc.xa.client.OracleXADataSource"
+		URL="jdbc:oracle:thin:@localhost:1521:xe"
+		user="testiaf_user"
 		password="testiaf_user"
-		driverClassName="oracle.jdbc.driver.OracleDriver"
-		url="jdbc:oracle:thin:@localhost:1521:xe"
-		maxActive="50"
```

Comment:
De maxActive is wel belangrijk, anders starten sommige adapter wanneer er meer database adapters zijn dan beschikbare connecties

Reply:
maxActive is een setting van oracle.jdbc.driver.OracleDriver. Die zit niet op oracle.jdbc.xa.client.OracleXADataSource